### DISCUSSION TITLE
Double Confirmations For Deleting + lock first admin user from accidental deletion

### DISCUSSION THREAD

**Discussion by cloudrack-ca** [Paymenter/Paymenter#179] [Ideas]
- adding a popup window to state double confirmations for deleting a resource category or client/user
- lock the admin user so that said user can not be accidentally deleted.

**Comment by finnjosten** [comment]
Maybe do it so it locks the last standing admin account, not the first as some want to remove it but maybe check if there is only 1 admin account and lock that one.

**Comment by cloudrack-ca** [comment]
> Maybe do it so it locks the last standing admin account, not the first as some want to remove it but maybe check if there is only 1 admin account and lock that one. 

sounds proper to me
